edge/pkg/servicebus/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"sync"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/edgecore/v1alpha1"
+)
+
+var Config Configure
+var once sync.Once
+
+type Configure struct {
+	v1alpha1.ServiceBus
+}
+
+func InitConfigure(sb *v1alpha1.ServiceBus) {
```

Comment:
```suggestion
func InitConfigure(s *v1alpha1.ServiceBus) {
```
`s` is fine here....